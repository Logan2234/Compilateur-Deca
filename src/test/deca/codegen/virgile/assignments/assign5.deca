// Created by Virgile, 10/01/2022
// test assignments.
{
    float a = 0 + 0.3;
    print(a);
}