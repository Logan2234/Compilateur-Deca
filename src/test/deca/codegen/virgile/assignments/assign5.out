  ** IMA ** ERREUR ** Ligne 6 : 
    ADD avec op1 : flottant et op2 : entier
